editor/src/clj/editor/targets.clj

```diff
@@ -357,7 +372,9 @@
                    ssdp-options (mapv target-option @ssdp-targets)]
                (cond
                  (seq launched-options)
-                 (into launched-options (concat [separator] ssdp-options))
+                 (if (> (count launched-options) 1)
+                 (into [{:label "All Launched Instances" :check true :command :target :user-data {:id :all-launched-targets}}] (concat launched-options [separator] ssdp-options))
```

Comment:
I think this is a good name, but if you don't think so, gimme your options.

Reply:
I agree. It's good!